l10n_it_withholding_tax/models/account.py

```diff
@@ -115,11 +115,12 @@ def generate_wt_moves(self):
         line_payment_ids.append(self.debit_move_id.id)
         line_payment_ids.append(self.credit_move_id.id)
         domain = [("id", "in", line_payment_ids)]
-        rec_lines = self.env["account.move.line"].search(domain)
+        rec_line_model = self.env["account.move.line"]
+        rec_lines = rec_line_model.search(domain)
 
         # Search statements of competence
         wt_statements = False
```

Comment:
`wt_statements` potrebbe avere lo stesso problema? più sotto alla linea 136 viene usata in un for...

Reply:
Grazie della review!
Sì potrebbe, ho corretto quello e un'altra inizializzazione che avrebbe potuto creare problemi, puoi controllare se ora va meglio?